qa/python_models/bls_memory/model.py

```diff
@@ -56,7 +56,7 @@ def _send_identity_tensor(self, size, is_decoupled):
         return input0_np, infer_response
 
     def test_bls_out_of_memory(self):
-        tensor_size = 1024 * 1024 * 1024
```

Comment:
Is it still going to be more than 1GB after changing it to 256?

Reply:
I believe the unit is in float so it's 4*256*1024*1024 bytes which is still larger than 1GB.